crates/common/types/blobs_bundle.rs

```diff
@@ -196,11 +193,14 @@ impl AddAssign for BlobsBundle {
     }
 }
 
+#[cfg(feature = "c-kzg")]
```

Comment:
I think we should move `c-kzg` stuff to a module and mark only that module

Reply:
I moved all the constants inside tests